src/GitHubApi.php

```
@@ -0,0 +1,56 @@
+
+	public function getInstallationIds( string $jwtToken ): array {
+        
+		$request = $this->httpFactory->create( "https://api.github.com/app/installations" );
+        $request->setHeader('Authorization', "Bearer $jwtToken");
+        $request->setHeader('Accept', "application/vnd.github+json");
+        $request->setHeader('User-Agent', "MediaWiki-ExternalContent");
+		$status = $request->execute();
+        return $status->isOK() ? json_decode( $request->getContent(), true ) : [];        
+	}
+
+	public function fetchAccessToken( string $installationId, string $jwtToken, $orgName ): string {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$orgName'. (undefined)

(UnusedFormalParameter)